index.js

```diff
@@ -1,53 +1,42 @@
 'use strict';
-var execFile = require('child_process').execFile;
-var spawn = require('child_process').spawn;
-var path = require('path');
-var multiTypeof = require('multi-typeof');
-var Promise = require('pinkie-promise');
+const path = require('path');
+const execa = require('execa');
+const multiTypeof = require('multi-typeof');
 
-function checkInput(input) {
+const checkInput = input => {
 	if (!multiTypeof(input, ['string', 'array'])) {
 		throw new TypeError('Expected a string or an array as input');
 	}
 
 	if (!Array.isArray(input)) {
-		var reg = new RegExp(/[^\s"']+|"([^"]*)"|'([^']*)'/g);
+		const reg = /[^\s"']+|"([^"]*)"|'([^']*)'/g;
+
 		input = input.match(reg);
 
-		input.forEach(function (el) {
-			input[input.indexOf(el)] = el.replace(/"/g, '');
-		});
+		for (const x of input) {
+			input[input.indexOf(x)] = x.replace(/"/g, '');
+		}
 	}
 
 	return input;
-}
-
-module.exports = function (input, opts) {
-	opts = opts || {};
+};
 
+module.exports = (input, opts) => {
 	if (process.platform !== 'win32') {
 		return Promise.reject(new Error('Only Windows systems are supported'));
 	}
 
-	return new Promise(function (resolve, reject) {
-		execFile(path.join(__dirname, 'nircmd.exe'), checkInput(input), opts, function (err, res) {
-			// NirCmd exits with this weird code even though it worked
-			if (err && err.code !== 4207175) {
-				reject(err);
-				return;
-			}
-
-			resolve(res);
-		});
+	return execa(path.join(__dirname, 'nircmd.exe'), checkInput(input), opts).catch(err => {
```

Comment:
Shouldn't this be `execa.stdout`? Now it returns an object with stdout and stderr.